mathics/builtin/graphics.py

```diff
@@ -2598,8 +2601,8 @@ def to_svg(self):
         content = self.content.boxes_to_xml(evaluation=self.graphics.evaluation)
         style = create_css(font_color=self.color)
         svg = (
-            '<foreignObject x="%f" y="%f" ox="%f" oy="%f" style="%s">'
-            "<math>%s</math></foreignObject>"
+            '<foreignObject x="%f" y="%f" ox="%f" oy="%f" width="100" height="100" style="%s">'
```

Comment:
The magic width and height values 100 should probably be options that can be set to 100 by default.

What do you think?

Reply:
I just added these parameters to see if it was a problem of boundaries. It wasn't...